pipelines/api/utils.py

```diff
@@ -77,12 +79,25 @@ def _slugify_file(filename):
     basename = filename.rsplit('/', 1)[-1]
     return basename.rsplit('.', 1)[0]
 
-
 def _run_id_iterator(slug):
```

Comment:
if we don't use `_run_id_iterator` anywhere else, suggest to use this name as the new function directly